sw/device/silicon_creator/rom_ext/rom_ext_start.S

```diff
@@ -35,54 +35,20 @@
    * More information about Ibex's interrupts can be found here:
    *   https://ibex-core.readthedocs.io/en/latest/03_reference/exception_interrupts.html
    */
-  .balign 256
+  .balignl 256, 0xc0001073
```

Comment:
nit: Can you add more details as to what this constant is?

Reply:
Done.